fairroot/geobase/FairGeoRootBuilder.h

```diff
@@ -10,30 +10,29 @@
 
 #include "FairGeoBuilder.h"   // for FairGeoBuilder
 
-#include <Rtypes.h>   // for FairGeoRootBuilder::Class, etc
+#include <TGeoManager.h>
 
 class FairGeoMedium;
 class FairGeoNode;
-class TGeoManager;
 
 /**
  * class to create geometry in ROOT
+ * \ingroup geobase
  * @author M. Al-Turany
  */
 class FairGeoRootBuilder : public FairGeoBuilder
 {
   private:
     FairGeoRootBuilder(const FairGeoRootBuilder&);
     FairGeoRootBuilder& operator=(const FairGeoRootBuilder&);
-
-  protected:
-    TGeoManager* geoManager;   // ROOT geometry manager
+    std::unique_ptr<TGeoManager> geoManager;   //!< ROOT geometry manager
 
   public:
     FairGeoRootBuilder();
     FairGeoRootBuilder(const char*, const char*);
+    FairGeoRootBuilder(const char*, const char*, std::unique_ptr<TGeoManager>);
     ~FairGeoRootBuilder() override = default;
-    void setGeoManager(TGeoManager* me) { geoManager = me; }
+    [[deprecated]] void setGeoManager(TGeoManager* me) { geoManager.reset(me); }
```

Comment:
Should this deprecation be added to the editorial changelog?

Reply:
I really hope that this code is actually only "internal FairRoot code". So I wanted to keep the amount of "paper work" to a minimum (I was actually thinking of just removing it at all).